Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -2,7 +2,7 @@
 
 ## Scope
 
-This document, the Software Carbon Intensity technical specification, describes a methodology for calculating the rate of carbon emissions for a software system. The purpose is to help users and developers make informed choices about which tools, approaches, architectures, and services they can use to minimize carbon emissions.
+This document, the Software Carbon Intensity technical specification, describes a methodology for calculating the rate of carbon emissions for a software system. The purpose of the score is to increase awareness and transparency of an applications sustainability credentials; to help users make informed choices about the applications they use in their daily lives (at home and at work). The score will also help application developers make better, evidence-based, decisions during system design, that will ultimately minimize carbon emissions. A reliable, consistent, fair and comparable measure allows targets to be defined during development and progress to be tracked.
```

Comment:
```suggestion
This document, the Software Carbon Intensity technical specification, describes a methodology for calculating the rate of carbon emissions for a software system. The purpose of the score is to increase awareness and transparency of an applications sustainability credentials. The score will help software practitioners make better, evidence-based, decisions during system design, development, and deployment, that will ultimately minimize carbon emissions. A reliable, consistent, fair and comparable measure allows targets to be defined during development and progress to be tracked.
```

Reply:
Removed users because we haven't comprehensively thought about their use of the spec.